app/portainer/views/init/admin/initAdmin.html

```diff
@@ -54,16 +54,10 @@
               <div class="col-sm-8">
                 <div class="input-group">
                   <input type="password" class="form-control" ng-model="formValues.ConfirmPassword" id="confirm_password" />
-                  <span class="input-group-addon"
-                    ><i
-                      ng-class="
-                        { true: 'fa fa-check green-icon', false: 'fa fa-times red-icon' }[
-                          form.password.$viewValue !== '' && form.password.$viewValue === formValues.ConfirmPassword
-                        ]
-                      "
-                      aria-hidden="true"
-                    ></i
-                  ></span>
+                  <span class="input-group-addon !rounded-r-[5px]">
```

Comment:
There's a conflicting style that used to give completely square corners, and the 5px is to match the `.input-group-addon` rounded corners. 

It could be worth playing around with the selector for this conflicting style, but I didn't want to get too far out of scope for this ticket.